new r_eff parametrization and update 2-moment formulas radarrefl and … (#378)
CliMA/CloudMicrophysics.jl: docs/src/Microphysics2M.md

\end{equation}
```

To obtain the logarithmic radar reflectivity ``L_z``, which is commonly used to refer to the radar reflectivity values, we sum ``Z_c`` and ``Z_r``, and we normalize the result with the equivalent return of a ``1 mm`` drop in a volume of a meter cube (``Z_0``). 
Then, we apply the decimal logarithm, and multiply the result by ``10``:
```math
\begin{equation}

> L_z = 10 \log_{10}(\frac{Z_c + Z_r}{Z_0}).

Reviewer: ```suggestion
L_z = 10 \log_{10}\left(\frac{Z_c + Z_r}{Z_0}\right).
```

\end{equation}
```
The resulting logarithmic dimensionless unit is decibel relative to ``Z``, or ``dBZ``.

### Effective radius

> The effective radius of hydrometeors (``r_{eff}``) is the weighted average of their size distribution, and it is defined as the ratio of the third  to the second moment of a droplet size distribution.

Reviewer: ```suggestion
The effective radius of hydrometeors (``r_{eff}``) is the area weighted radius of the population of particles.  It can be computed as the ratio of the third  to the second moment of the size distribution.
```

In our case, since we have two separate distributions for cloud and rain, after applying the appropriate transformation to express the distributions as functions of the radius, we obtain:
```math
\begin{equation}

> r_{eff} = \frac{(M_{c}^3 + M_{r}^3)}{M_{c}^2 + M_{r}^2} = \frac{{\int_0^\infty r^{5} \, (f_c(r) + f_r(r)) \, dr}}{{\int_0^\infty r^{4} \, (f_c(r) + f_r(r)) \, dr}}.

Reviewer: Why r^5 and r^4 and not r^3 and r^2?

Reviewer: Also, I think we were using `n(r)` for the particle size distribution. Could we replace `f(r)` by that?

Author: It's r^4, and r^5 because when going from the mass distribution (proportional to r^3) to the radius one, I have to multiply by the derivative of the mass wrt r (= 4 * pi * r^2 * \rho)

|------------|---------------------------|
|``A``       | ``4.7``                   |

> ### Effective radius - Liu and Hallett, 1997

Reviewer: Could we move that to the same section as the effective radius computed from the size distribution?

An additional option for the parametrization of the effective radius is obtained following [Liu1997](@cite).
In this case 
```math
\begin{equation}
  r_{eff} = \frac{r_{vol}}{k^{\frac{1}{3}}},
\end{equation}
```
where:
  - ``r_{vol}`` represents the volume-averaged radius,
  - ``k = 0.8``. 

Where the volume-averaged radius is computed using
```math
\begin{equation}
  r_{vol} = \left(\frac{3}{4 \pi \rho_w}\right)^{\frac{1}{3}} \, \left(\frac{L}{N}\right)^{\frac{1}{3}} = \left(\frac{3  \rho (q_{liq} + q_{rai})}{4 \pi \rho_w (N_{liq} + N_{rai})}\right)^{\frac{1}{3}},
\end{equation}
```
where:

>   - ``L = \rho q_{liq}``, is the liquid water content,

Reviewer: ```suggestion
  - ``L = \rho q``, is the water content,
```
Since we are taking both cloud liquid and rain contributions?
